beacon_node/eth2_libp2p/src/rpc/mod.rs

```diff
@@ -97,7 +102,25 @@ pub struct RPC<TSpec: EthSpec> {
 impl<TSpec: EthSpec> RPC<TSpec> {
     pub fn new(log: slog::Logger) -> Self {
         let log = log.new(o!("service" => "libp2p_rpc"));
+        let limiter = RateLimiterBuilder::new()
+            .one_every(Protocol::MetaData, Duration::from_secs(5))
+            .one_every(Protocol::Ping, Duration::from_secs(15))
```

Comment:
```suggestion
            .one_every(Protocol::Ping, Duration::from_secs(5))
```

I think Teku ping us every 5 seconds, so lets allow them